tests/cpp/analysis/bls_analyzer_test.cpp

```diff
@@ -38,8 +25,8 @@ class BLSAnalyzerTest : public ::testing::Test {
     child_snode_ = &(parent_snode_->insert_children(SNodeType::place));
     child_snode_->dt = PrimitiveType::i32;
 
-    TestStructCompiler sc;
-    sc.infer_snode_properties(*root_snode_);
+    FakeStructCompiler sc;
+    sc.run(*root_snode_, /*unused=*/false);
```

Comment:
nit: Do not use `/*unused=*/` which seems to me that there's a field called `unused`? Maybe `/*(unused)*/` is easier to understand here.

Reply:
Ack. I added another `run(SNode& root)` method so that the callers don't have to always annotate with `/*(unused)*/`